src/hotspot/os_cpu/windows_aarch64/vm_version_windows_aarch64.cpp

```diff
@@ -97,3 +97,8 @@ void VM_Version::get_os_cpu_info() {
     }
   }
 }
+
+void VM_Version::get_compatible_board(char *buf, int buflen) {
+  assert(buf != NULL, "invalid argument");
+  *buf = '\0';
```

Comment:
This is wrong too: it should check buflen.

Reply:
I added assert for buflen in new commit.